soh/soh/GameMenuBar.cpp

```diff
@@ -617,6 +617,15 @@ namespace GameMenuBar {
                         ImGui::EndMenu();
                     }
 
+                    UIWidgets::Spacer(0);
+
+                    bool disabled = !CVarGetInteger("gSuperSonic", 0);
+                    const char* disabledTooltip = "This option is disabled because \"Super Sonic Mode\" is turned off";
+                    UIWidgets::PaddedEnhancementCheckbox("Super Sonic Mode", "gSuperSonic", true, false);
+                    UIWidgets::Tooltip("Rupees reduced over time, Link dies when the count hits 0.");
+                    UIWidgets::PaddedEnhancementSliderInt("Super Sonic Interval: %d", "##SsmInterval", "gSsmInterval", 3, 5, "", 5, false, true, false, disabled, disabledTooltip);
```

Comment:
```suggestion
                    UIWidgets::PaddedEnhancementSliderInt("Super Sonic Interval: %d", "##SsmInterval", "gSsmInterval", 3, 5, "", 5, false, true, false, !CVarGetInteger("gSuperSonic", 0), "This option is disabled because \"Super Sonic Mode\" is turned off");
```

Reply:
I've used variables for other settings because it's much easier to read the text for the tooltip in code that way, especially when the tooltips become longer.